core/src/ast/attrs.rs

```diff
@@ -38,6 +39,9 @@ pub struct Attrs {
     /// not see the impl blocks.
     pub attrs: Vec<DiplomatBackendAttr>,
 
+    /// Backend specific configuration attributes:
+    pub config_attrs: Vec<DiplomatBackendConfigAttr>,
```

Comment:
Thought: these shouldn't be allowed on random types, we should require they go on a toplevel type in lib.rs (with whatever name they get).

Reply:
(still an issue)